crates/sui-graphql-rpc/src/extensions/query_limits_checker.rs

```diff
@@ -185,9 +203,16 @@ impl Extension for QueryLimitsChecker {
                 sel_set,
                 &mut running_costs,
                 variables,
+                ctx,
             )?;
             max_depth_seen = max_depth_seen.max(running_costs.depth);
         }
+        if let Some(timer) = latency_timer {
+            timer.observe_duration();
+        }
+        if let Some(timer) = timer_latency_by_path {
+            timer.observe_duration();
+        }
```

Comment:
What's the difference between this and the other latency timer? They get instantiated and updated at the same level - I was assuming that we update the path timer within the for loop or something

Reply:
The high level idea was to record a metric by query path (some sort of normalized path), similarly to what we do today with JSON RPC (each endpoint has its own "metric" values).